tsconfig.base.json

```diff
@@ -1,24 +1,35 @@
 {
 	"compilerOptions": {
-		"strict": true,
-		"moduleResolution": "node",
-		"module": "commonjs",
-		"target": "es2020",
-		"lib": ["esnext", "esnext.array", "esnext.asynciterable", "esnext.intl", "esnext.symbol"],
+		"allowSyntheticDefaultImports": true,
+		"alwaysStrict": true,
 		"declaration": true,
 		"declarationMap": true,
-		"sourceMap": true,
-		"removeComments": false,
-		"alwaysStrict": true,
-		"pretty": true,
+		"emitDecoratorMetadata": true,
+		"esModuleInterop": true,
+		"experimentalDecorators": true,
+		"forceConsistentCasingInFileNames": true,
+		"importHelpers": true,
+		"importsNotUsedAsValues": "error",
 		"incremental": true,
+		"lib": ["esnext"],
+		"module": "commonjs",
+		"moduleResolution": "node",
+		"newLine": "lf",
 		"noEmitHelpers": true,
-		"importHelpers": true,
-		"skipLibCheck": true,
-		"esModuleInterop": true,
+		"noFallthroughCasesInSwitch": true,
+		"noImplicitOverride": true,
+		"noImplicitReturns": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
 		"preserveConstEnums": true,
+		"pretty": true,
```

Comment:
Why?

Reply:
`Enable color and formatting in output to make compiler errors easier to read`

Probably redundant, can remove